demos/demo-simple-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -6,4 +6,9 @@
 
   <display-name>Simple Web Application</display-name>
 
+  <mime-mapping>
+    <extension>webp</extension>
+    <mime-type>image/webp</mime-type>
+  </mime-mapping>
+
```

Comment:
This should not be required now that the server's MIME mapping includes `webp`.

I understand this is only for files with non-standard extensions that applications wants the server to send with the proper `Content-Type` header, but `webp` is now a standard one.

I would remove it so that we can verify the standard server behavior.

Reply:
I need to change this.

The problem arises from the combination of a webapp specific mime-type (one that does not exist in `MimeTypes`) used with `GzipHandler`.